resources/abi/README.md

```diff
@@ -0,0 +1,74 @@
+# ABI change monitoring
+
+When the OpenAssetIO C++ ABI changes, we need to know about it to inform
+our release notes and semver versioning.
+
+This is aided (though not solved!) by automated testing against a
+snapshot dump of the ABI. The tool we use for this is [libabigail](https://sourceware.org/libabigail/manual/libabigail-overview.html),
+in particular its `abidiff` and `abidw` tools.
+
+> **Note**
+>
+> The ABI on other platforms, including other Linux distributions, may
+> (subtly) vary. Always generate a dump of a build from the same Docker
+> image as is used on CI, i.e. `ghcr.io/openassetio/openassetio-build`.
+
+If the ABI of an OpenAssetIO library changes, and the change is
+unavoidable, then a new XML snapshot dump of the ABI must be created
+(and release notes updated as appropriate).
+
+To do this we use `abidw`. For convenience, the `openassetio-build`
+Docker image has libabigail pre-installed.
+
+The generated snapshot is then checked against new builds using
+`abidiff`.
+
+## Example of updating the ABI snapshot
```

Comment:
[Consider][Minor] Maybe this could more clearly be a set of steps you need to follow with an instructive tone. Ie, rather than 
> This will generate a new XML formatted ABI dump to overwrite the
existing XML dump under `resources/abi`.

do

> Generate a new XML formatted ABI dump and overwrite the existing XML dump under `resources/abi`

A couple of other things on this note. Perhaps the title could be "Updating the ABI snapshot" rather than "example of", as well as potentially presenting this as a numbered list of instructions, rather than an example. Not much changes to content, just presentation.

Just cause the goal for any process should be being able to do it on autopilot.

Reply:
Good idea, restructured in 99e93a2c